code/controllers/subsystem/profiler.dm

```diff
@@ -73,7 +73,7 @@ SUBSYSTEM_DEF(profiler)
 	// FETCH MAPTICK PROFILE //
 
 	// Fetch info
-	var/current_sendmaps_data = world.Profile(PROFILE_REFRESH, type = "sendmaps", format = "json")
+	var/current_sendmaps_data = world.Profile(PROFILE_REFRESH, "sendmaps", "json")
```

Comment:
```suggestion
	var/current_sendmaps_data = world.Profile(PROFILE_REFRESH, "sendmaps", "json")
```

I can't remember why I did this but it might need reverting

Reply:
This was done because of illegal use of named args, this proc doesn't need named args.
https://www.byond.com/docs/ref/#/world/proc/Profile